libs/ui-lib-tests/cypress/views/customManifestsPage.ts

```diff
@@ -0,0 +1,35 @@
+export const customManifestsPage = {
+  getFileName: () => {
+    return cy.get('[data-testid=filename-0]');
+  },
+  getFileNameError: () => {
+    return cy.get('div.pf-c-form__helper-text.pf-m-error');
+  },
+  getStartFromScratch: () => {
+    return cy.get('.pf-c-file-upload .pf-c-empty-state__secondary > button');
```

Comment:
Needs a data-testid at least or use findByRole

Reply:
This button comes from CodeEditor component from patternfly. At least is easier to get with CSS selector.